src/components/QuranReader/ContextMenu/styles/ContextMenu.module.scss

```diff
@@ -100,6 +105,51 @@ $pixel: 1px;
   }
 }
 
+.readingPreferenceContainer {
```

Comment:
2 issues:

<img width="1726" height="892" alt="Image" src="https://github.com/user-attachments/assets/ec40fd05-3857-4b97-9bb9-d05a30f5ce5d" />


1. Settings is not aligned on the same line as the Surahs navigation drawer on the left.
2. The buttons to navigate up and down are on top of the settings drawer when it's open.